website/routes.py

```diff
@@ -430,6 +431,11 @@ def make_url_map(app):
         ),
     ])
 
+    process_rules(app, [
+        Rule('/explore/activity/popular/raw/', 'get', discovery_views.popular_activity_json, json_renderer),
+
+    ], prefix='/api/v1')
```

Comment:
I was instructed to make this a v1 route.   I basically want it to return a portion of the data used for another v1 route, /explore/activity/.  It's a one-off thing that they didn't want to make a v2 endpoint.